File: class4/ex13_positiveNegativeZero.c
/*Faça um programa que leia um número inteiro e informe se ele é
positivo, negativo ou zero.*/

#include <stdio.h>

int main() {

    int n;

    printf("Digite um número inteiro: ");
    scanf("%d", &n);

    if(n < 0) {

        printf("O número %d é negativo.\n", n);
    }
    else if(n >= 0) {

        printf("O número %d é positivo.\n", n);
    }
    else {

        printf("O número %d é zero.\n", n);
    }

    return 0;
}

File: GO/class4/ex13_positiveNegativeZero.go
/*Faça um programa que leia um número inteiro e informe se ele é
positivo, negativo ou zero.*/

package main

import "fmt"

func main() {

	var n int

	fmt.Printf("Digite um número inteiro: ")
	fmt.Scan(&n)

	if n < 0 {

		fmt.Printf("O número %d é negativo.\n", n)
	} else if n > 0 {

		fmt.Printf("O número %d é positivo.\n", n)
	} else {

		fmt.Printf("O número %d é zero.\n", n)
	}
}
